docs/Getting-Started/Run-Docker-Image.md

```diff
@@ -144,7 +145,11 @@ Where `mygenesis.json` is your custom configuration file and `path` is the absol
 
 ## Permissions Configuration File 
 
-Specify a permissions configuration file. This is equivalent to specifying the `--permissions-config-file` option.
+Specify a permissions configuration file. This is equivalent to specifying the `--permissions-accounts-config-file` 
```

Comment:
you write about two options but to me they are the same `--permissions-accounts-config-file` and `--permissions-accounts-config-file`

Reply:
Good point - fixed the second node to be nodes.